README.rst

```diff
@@ -294,6 +294,51 @@ global settings tree, but have to be separatedly accessed using the ``bluetooth`
     Patches improving this are welcome!
 
 
+Group Control
+--------------
+
+.. code-block::
+
+    $ songpal group 
+    
+Group command require using the PNP URL ``--url`` instead of the API ``--endpoint``, and you'll need the ``UUIDs`` of the devices you want to group as well. All group commands should be executed on the master
```

Comment:
```suggestion
Group command require using the UPnP URL ``--url`` instead of the API ``--endpoint``, and you'll need the ``UUIDs`` of the devices you want to group as well. All group commands should be executed on the master
```

Reply:
Probably worth mentioning here also, that you can obtain these (UDNs/UUIDs) and the baseurl for upnp endpoint using `songpal discover`?

Reply:
One question, does the state information show anything about the master session (when called on slaves)? If so, the code could be adapted to allow executing commands also on slave instances.